README.md

```diff
@@ -87,11 +87,34 @@ Provide the configuration endpoint ([AWS Reference](http://docs.aws.amazon.com/A
 
 ```
 
-Multiple clusters Auto Discovery
-===============================
+If instead of using ElastiCache a custom auto discovery is needed you can provide your own function (specified as a MFA) that returns `{ok, list({Host :: inet:hostname(), Port :: inet:port_number()})}` to be used. Example:
+```erlang
+    [{cluster_b,
+        [{servers, {mfa, {my_module, my_function, []}}},
+             {sharding_algorithm, {mero, shard_crc32}},
+             {workers_per_shard, 1},
+             {pool_worker_module, mero_wrk_tcp_binary}]
+    },
+    ...
+]
 
-We also support the setup of multiple pyhisical clusters with autodiscovery assigned to the same logical cluster.
-It could be the case in which some of these pyhisical clusters perform better than others, in which you can use a third argument called the ClusterSpeedFactor which has to be a small integer. The default `ClusterSpeedFactor` is `1`.
+```
+
+```erlang
+-module(my_module).
+
+-export([my_function/0]).
+
+my_fFunction() ->
```

Comment:
```suggestion
my_function() ->
```
😉

Reply:
Sorry, fixed in next commit :sweat_smile: